### PR TITLE
feat: Bump aries-framework-go v0.1.6 to v0.1.8

### LINKED ISSUE #9: Bump aries-framework-go version `v0.1.6` to `v0.1.8`
## Background

Bump aries-framework-go version v0.1.6 to v0.1.8. You can refer to the change log in this [link](https://github.com/hyperledger/aries-framework-go/blob/main/CHANGELOG.md).

## Implement
When applying the v0.1.8 and running the test code, it returns fail. Because the aries-framework-go used a JSON-LD Context API after v0.1.7. So we need to implement a `DocumentLoader` for credential as following code:
```go
func NewFrameWork() (*FrameWork, error) {
	storeProvider := mem.NewProvider()
	contextStore, err := ldstore.NewContextStore(storeProvider)
	if err != nil {
		return nil, err
	}

	remoteProviderStore, err := ldstore.NewRemoteProviderStore(storeProvider)
	if err != nil {
		return nil, err
	}

	ctx, err := context.New(
		context.WithJSONLDContextStore(contextStore),
		context.WithJSONLDRemoteProviderStore(remoteProviderStore),
	)
	if err != nil {
		return nil, err
	}

	loader, err := ld.NewDocumentLoader(
		ctx,
		ld.WithRemoteDocumentLoader(jsonld.NewDefaultDocumentLoader(&http.Client{})),
	)
	if err != nil {
		return nil, err
	}

	return &FrameWork{
		loader: loader,
	}, nil
}
```
So, when the user wants to use `SignCredential`, `VerifyPresentation` or other method, the user should need to declare an variable with above code.

### PR DESCRIPTION
**Pull request by inchori**
Closes #9

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -31,8 +31,8 @@ jobs:
       - name: Test
         run: make test
 
-      - name: Build Android
-        run: make build-android
+#      - name: Build Android
+#        run: make build-android
```
Comment (inchori) [inline .github/workflows/ci.yaml:35]: The aries-framework v0.1.8 used a `github.com/google/tink/go v0.1.6`, but needs to use a `github.com/google/tink/go v0.1.7` for the gomobile bind. So for now, I'll comment this part in CI.

File: pkg/vc/command.go [hunk 2]
```diff
@@ -3,36 +3,40 @@ package vc
...
 	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/suite"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/suite/bbsblssignature2020"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/suite/ecdsasecp256k1signature2019"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
 )
 
 // SignCredential creates a verifiable credential by adding a proof to the credential.
-func SignCredential(credential []byte, privKey []byte, opts *ProofOptions) ([]byte, error) {
-	cred, err := verifiable.ParseCredential(credential, verifiable.WithDisabledProofCheck())
+func (f *FrameWork) SignCredential(credential []byte, privKey []byte, opts *ProofOptions) ([]byte, error) {
```
Comment (youngjoon-lee) [inline pkg/vc/command.go:19]:
```suggestion
func (f *Framework) SignCredential(credential []byte, privKey []byte, opts *ProofOptions) ([]byte, error) {
```
lowercases :)

File: pkg/vc/command.go [hunk 3]
```diff
@@ -163,7 +169,7 @@ const (
 	EcdsaSecp256k1Signature2019 = "EcdsaSecp256k1Signature2019"
 )
 
-func addProof(provableData provable, privKey []byte, opts *ProofOptions) error {
+func addProof(provableData provable, privKey []byte, opts *ProofOptions, loader *ld.DocumentLoader) error {
```
Comment (youngjoon-lee) [inline pkg/vc/command.go:172]:
```suggestion
func (f *Framework) addProof(provableData provable, privKey []byte, opts *ProofOptions) error {
```
You can also embed this function to the `Framework`

**Review by gyuguen — APPROVED** [review]
LGTM

**Review by audtlr24 — APPROVED** [review]
lgtm

**Pull request merged**